<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XinxListWidget</class>
 <widget class="QWidget" name="XinxListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="m_list"/>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_btnAdd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset resource="../xinxpluginselector.qrc">
         <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnDel">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../xinxpluginselector.qrc">
         <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnDef">
       <property name="text">
        <string>&amp;Default</string>
       </property>
       <property name="icon">
        <iconset resource="../xinxpluginselector.qrc">
         <normaloff>:/images/checkbox.png</normaloff>:/images/checkbox.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnUp">
       <property name="text">
        <string>Move &amp;Up</string>
       </property>
       <property name="icon">
        <iconset resource="../xinxpluginselector.qrc">
         <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnDown">
       <property name="text">
        <string>Move &amp;Down</string>
       </property>
       <property name="icon">
        <iconset resource="../xinxpluginselector.qrc">
         <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_list</tabstop>
  <tabstop>m_btnAdd</tabstop>
  <tabstop>m_btnDel</tabstop>
  <tabstop>m_btnDef</tabstop>
 </tabstops>
 <resources>
  <include location="../xinxpluginselector.qrc"/>
 </resources>
 <connections/>
</ui>
